### ISSUE TITLE
Add emoji support to all created fields

### ISSUE THREAD

**Issue by privatemaker** [nextcloud/forms#1717]
**Is your feature request related to a problem?**

Emojis make all sorts of text more human 💡 With Forms emojis allow the author to make their forms more fun, creative, and engaging 🌱 👌🏻 🕺🏻 🥳 

Currently, emojis render correctly in all areas of Forms, but the author of a Form must copy and paste emojis manually from another source. This is time consuming and undesirable ☹️ 

**Describe the solution you'd like**

Incorporate a picker icon in form fields as seen here in Talk.

![talk-emoji-picker](https://github.com/nextcloud/forms/assets/49612519/140eeac4-ec4c-4e1b-83a3-e525f0bc3a1f)

And also support the typing trigger `:s...` which autosuggests `:smile` 😄

**Comment by Chartman123** [comment]
One problem could be with the export as CSV, other than that I think this is nice to have but not essential for the function of Forms

**Comment by susnux** [comment]
I do not think this would be a problem, but I personally do not like that RichText component as the support is somewhat limited and broken.
There are many browser bugs that interfere with content editable.

We reverted it currently here in forms because it broke the workflow for many users.
As I do not think that feature has high priority I would postpone it (on mobile you probably always have an emoji keyboard available provided by your OS).
